js/src/io/parquet/async.rs

```diff
@@ -117,6 +118,22 @@ impl ParquetFile {
         let (schema, batches) = table.into_inner();
         Ok(Table::new(schema, batches))
     }
+    #[wasm_bindgen]
+    pub async fn read_stream(
+        &self,
+        options: JsValue,
+    ) -> WasmResult<wasm_streams::readable::sys::ReadableStream> {
+        let options: Option<JsParquetReaderOptions> = serde_wasm_bindgen::from_value(options)?;
+        let stream = self.file.read_stream(options.unwrap_or_default().into())?;
+        let out_stream = stream
+            .map(|maybe_batch| {
+                let batch = maybe_batch.unwrap();
```

Comment:
Is there any way to not unwrap here?

Reply:
Definitely, I suspect it's a matter of ensuring the errors are neutered (they have a nasty habit of dragging in references to Non-send aspects of the original stream/futures), should have a change up for that shortly.